app/components/filter-distance-from-point.js

```diff
@@ -1,41 +1,69 @@
 import Component from '@ember/component';
-// import { argument } from '@ember-decorators/argument';
 import { action, computed } from '@ember-decorators/object';
 import { generateCircleFromFeet } from 'labs-zap-search/helpers/generate-circle-from-feet';
 
-// it takes a point and radius, computes a geojson circle
-// and handles clicks that query mapbox-gl for intersections
-// triggers optional events
+/**
+ * The FilterDistanceFromPoint component takes a point and radius, computes a geojson circle
+ * and handles click events that query mapbox-gl for intersections triggers optional events.
```

Comment:
im a bit confused by the language here. want to make sure I understand:  It handles click events (on what?), and defines the actions for those click events that query mapbox-gl for intersections with the computed circle, and then triggers optional events

Reply:
Handles click events on the mapbox-gl map itself. Yes. gonna revise for clarity!